static/app/components/dashboards/issueWidgetQueriesForm.tsx

```diff
@@ -93,6 +93,9 @@ export const SearchConditionsWrapper = styled('div')`
 
 const StyledIssueListSearchBar = styled(IssueListSearchBar)`
   flex-grow: 1;
+  button:not([aria-label='Clear search']) {
```

Comment:
Yes not rendering in React would be a better solution. I am reusing the search bar component from issues though, which doesn't have props to disable the buttons so I'm using this css shortcut to avoid impacting issues code for now. Could definitely update the search bar component in the future if needed though!